src/components/Radio/ReciterStationList.tsx

```diff
@@ -57,10 +39,14 @@ const ReciterStationList = ({ reciters }: ReciterStationListProps) => {
               stationId: reciter.id,
               type: StationType.Curated,
             });
-            playReciterStation(reciter, dispatch);
+            audioService.send({
+              type: 'PLAY_RADIO',
+              stationType: StationType.Reciter,
+              stationId: reciter.id,
+            });
           };
-        if (isSelectedStation && isAudioPlaying) onClick = () => triggerPauseAudio();
-        if (isSelectedStation && !isAudioPlaying) onClick = () => triggerPlayAudio(playbackRate);
+        if (isSelectedStation && isAudioPlaying) onClick = () => audioService.send('TOGGLE');
+        if (isSelectedStation && !isAudioPlaying) onClick = () => audioService.send('TOGGLE');
```

Comment:
this should be simplified, we are unnecessarily checking `isAudioPlaying` while if it's true or false the `onClick` value is the same